pkg/crc/preflight/preflight.go

```diff
@@ -74,11 +76,14 @@ func (check *PreflightCheck) doCheck() error {
 
 func (check *PreflightCheck) doFix() error {
 	if check.fixDescription == "" {
-		// warning, this is a programming error
-	} else {
-		logging.Infof("%s", check.fixDescription)
+		panic("Should not happen, empty description for fix")
```

Comment:
Same goes here, we should also put which config have missing `fixDescription` during this panic.